contenta_jsonapi.profile

```diff
@@ -15,4 +15,27 @@ use Drupal\Core\Form\FormStateInterface;
 function contenta_jsonapi_form_install_configure_form_alter(&$form, FormStateInterface $form_state) {
   // Add a value as example that one can choose an arbitrary site name.
   $form['site_information']['site_name']['#placeholder'] = t('Contenta JSON API');
+
+  $form['contenta_jsonapi'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Contenta settings'),
+    '#weight' => -10,
+  ];
+    
+  $form['contenta_jsonapi']['include_recipes_magazin'] = [
+    '#title' => t('Include the recipes magazin'),
```

Comment:
> 'Install the demo content'.

+1 to not couple to the concrete default content. People don't have to know about it.